subprojects/core/src/main/java/org/gradle/internal/fingerprint/impl/DefaultOutputFileCollectionFingerprinter.java

```diff
@@ -23,9 +23,9 @@
 import org.gradle.api.tasks.FileNormalizer;
 import org.gradle.internal.fingerprint.OutputNormalizer;
 
-public class DefaultOutputNormalizer extends AbstractPathOnlyFileCollectionFingerprinter implements OutputNormalizer {
+public class DefaultOutputFileCollectionFingerprinter extends AbstractPathOnlyFileCollectionFingerprinter implements OutputFileCollectionFingerprinter {
```

Comment:
Does it make sense to have a separate `OutputFileCollectionFingerprinter` interface for this? Same for all the other implementations. I think we could simplify the code quite a bit if we discarded the interfaces.

Reply:
I removed the interfaces.